api/swagger.yaml

```diff
@@ -4443,6 +4443,8 @@ paths:
       responses:
         200:
           description: "no error"
+        404:
+          description: "repository not found"
```

Comment:
@thaJeztah Yep, make information clear is much important. I've change the 404 status code description to ``` repository not exist or no read access```  to cover the both cases.